app/presenters/mfa_confirmation_presenter.rb

```diff
@@ -1,14 +1,23 @@
 class MfaConfirmationPresenter
-  def initialize(show_skip_additional_mfa_link: true)
+  def initialize(show_skip_additional_mfa_link: true, webauthn_platform_set_up_successful: false)
     @show_skip_additional_mfa_link = show_skip_additional_mfa_link
+    @webauthn_platform_set_up_successful = webauthn_platform_set_up_successful
   end
 
   def heading
-    I18n.t('titles.mfa_setup.suggest_second_mfa')
+    if webauthn_platform_set_up_successful?
+      I18n.t('titles.mfa_setup.face_touch_unlock_confirmation')
```

Comment:
Can we add some test coverage to the presenter spec for this new logic branch path?